Cargo.toml

```diff
@@ -20,7 +20,7 @@ structopt = "0.3"
 structopt-derive = "0.4"
 rustc-demangle = "0.1"
 cpp_demangle = "0.3"
-scroll = "0.10"
+scroll = "0.11"
```

Comment:
I was going to hold back on this since iirc, goblin 0.4 is still on 0.10, and this means there are two scrolls in the crate graph technically now. It's not super important to update this right now, I'd prefer to wait until goblin has the matching scroll.

Reply:
Should I edit the PR to revert this change?